_includes/about/about.scss

```diff
@@ -6,12 +6,17 @@
   .content-block {
     @include flex(row, center, flex-start);
 
+    @include small-desktop-screen {
+      padding: 0 10px;
+    }
+
     @include tablet-screen {
       @include flex(column, center, center);
       padding: 0 100px;
     }
 
     @include small-tablet-screen {
+      max-width: 460px !important;
```

Comment:
Can we handle it without `!important`?

Reply:
no, because we need to rewrite styles from style.scss